AppCenterAnalytics/AppCenterAnalyticsTests/MSEventPropertiesTests.m

```diff
@@ -109,4 +109,22 @@ - (void)testAppCenterCopyHas20PropertiesWhenSelfHasMoreThan20 {
   XCTAssertEqual([appCenterCopy.properties count], 20);
 }
 
+- (void)testEventPropertiesHaveValidPropertiesForAppCenterWhenSutIsCopiedForAppCenter {
```

Comment:
I dislike using `sut` in the title.  We can use it for the variable name since it's a convention but we should just use something like `testEventPropertiesHaveValidPropertiesForAppCenterWhenCopiedFromAppCenter`